python/pyspark/sql/tests/test_udtf.py

```diff
@@ -74,26 +74,23 @@ def eval(self):
                 yield "hello", "world"
 
         func = udtf(TestUDTF, returnType="c1: string, c2: string")
-        rows = func().collect()
-        self.assertEqual(rows, [Row(c1="hello", c2="world")])
+        assertDataFrameEqual(func(), [Row(c1="hello", c2="world")])
```

Comment:
If we prefer `assertDataFrameEqual`, is it possible to replace `assertEqual` with `assertDataFrameEqual` in initialization?

```
    self.assertEqual = assertDataFrameEqual
```

Reply:
That’s a wise alternative to minimize code changes. Since I also want to remove the “collect” in the test cases, I think replacing it with assertDataFrameEqual is more intuitive and straightforward.